docs/csharp/programming-guide/inside-a-program/coding-conventions.md

```diff
@@ -58,148 +57,181 @@ Good layout uses formatting to emphasize the structure of your code and to make
 
-     The following example uses explicit typing in a `foreach` statement.
+  The following example uses explicit typing in a `foreach` statement.
 
-     [!code-csharp[csProgGuideCodingConventions#12](../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs#12)]
+  :::code language="csharp" source="../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs" id="Snippet12":::
 
-     > [!NOTE]
-     > Be careful not to accidentally change a type of an element of the iterable collection. For example, it is easy to switch from <xref:System.Linq.IQueryable?displayProperty=nameWithType> to <xref:System.Collections.IEnumerable?displayProperty=nameWithType> in a `foreach` statement, which changes the execution of a query.
+  > [!NOTE]
+  > Be careful not to accidentally change a type of an element of the iterable collection. For example, it is easy to switch from <xref:System.Linq.IQueryable?displayProperty=nameWithType> to <xref:System.Collections.IEnumerable?displayProperty=nameWithType> in a `foreach` statement, which changes the execution of a query.
 
-### Unsigned Data Type  
+### Unsigned data types  
   
 In general, use `int` rather than unsigned types. The use of `int` is common throughout C#, and it is easier to interact with other libraries when you use `int`.  
   
 ### Arrays  
   
-Use the concise syntax when you initialize arrays on the declaration line.  
-  
-[!code-csharp[csProgGuideCodingConventions#13](../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs#13)]  
+Use the concise syntax when you initialize arrays on the declaration line. In the following example, note that you can't use `var` instead of `string[]`.  
   
-### Delegates  
+:::code language="csharp" source="../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs" id="Snippet13a":::
+
+If you use explicit instantiation, you can use `var`.
+
+:::code language="csharp" source="../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs" id="Snippet13b":::
+
+If you specify an array size, you have to initialize the elements one at a time.
   
-Use the concise syntax to create instances of a delegate type.  
+:::code language="csharp" source="../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs" id="Snippet13c":::
   
-[!code-csharp[csProgGuideCodingConventions#14](../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs#14)]  
+### Delegates  
   
-[!code-csharp[csProgGuideCodingConventions#15](../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csprogguidecodingconventions/cs/program.cs#15)]  
+Use the concise syntax to create instances of a delegate type. In the `Program` class, define the delegate type and a method that has a matching signature.  
```

Comment:
I wonder if we should recommend using `Func<>` and `Action<>` instead of defining delegate types. What do you think?